src/sass/formapp.scss

```diff
@@ -3,12 +3,67 @@ body {
   font-family: 'ProximaSoft', 'Droid Sans', 'helvetica neue', 'arial', sans-serif !important;
   padding: 24px;
 }
+.col-form-label {
+  font-size: 18px;
+  font-weight: bold;
+}
 .form-control:disabled, .form-control[readonly] {
   background-color: inherit;
   border: none !important;
 }
-.btn:disabled {
-  opacity: .5;
+.block {
```

Comment:
This seems like a bad idea.  `block` looks to be a utility class to make something into a block.

Reply:
actually I'm not sure where `.block` is coming from, is this a custom class unrelated to form.io / bootstrap?  If so a name other than `block` might be useful due to `display: block`